htmlhint-server/src/server.ts

```
@@ -60,6 +65,74 @@ function getRange(error: htmlhint.Error, lines: string[]): any {
     };
 }
 
+function loadRules(HTMLHint: any, force: boolean): any {
+    if (force) {
+        rulesLoaded = false;
+    }
+
+    if (rulesLoaded || htmlhint == null) {
+        return;
+    }
+    try {
+        if (!settings.htmlhint || !settings.htmlhint.rulesDir) {
+            return;
+        }
+        let rulesDir = settings.htmlhint.rulesDir;
+        let absoluteDir = '';
+        //check absolute path
+        let exists = fs.existsSync(rulesDir);
+        if (exists) {
+            absoluteDir = rulesDir;
+        } else {
+            //relative dir, find it based on workspace roots
+            absoluteDir = path.resolve(rootFolder, rulesDir);
+            exists = fs.existsSync(absoluteDir);
+        }
+
+        if (exists) {
+            //load all rules
+            loadCustomRules(absoluteDir, HTMLHint);
+        }
+    }
+    catch (e) { }
```

why are exceptions getting swallowed here?  Is this necessary?